src/python/cli_new/lib/cli/http.py

```diff
@@ -64,7 +65,7 @@ def read_endpoint(addr, endpoint, config, query=None):
                            .format(url=url, error=str(exception)))
 
 
-def get_json(addr, endpoint, config, condition=None, query=None):
+def get_json(addr, endpoint, config, query=None):
```

Comment:
Are you sure that nothing uses `condition`?

Also, if you remove t, please remove the mention in the docstring as well:
```
subject to the condition specified in 'condition'. If we are
```

Reply:
Not the mesos_cli. I had a look. But of course, I will also feel better u you keep an eye on it. :-)